pyproject.toml

```diff
@@ -287,9 +287,10 @@ external = ["V"]
 task-tags = ["NOTE", "TODO", "FIXME", "XXX"]
 
 [tool.ruff.format]
-quote-style = "double"
+docstring-code-format = true
 indent-style = "space"
 line-ending = "lf"
+quote-style = "double"
```

Comment:
Note: the rest of these settings (quote-style, line-ending, indent-style) seem to mostly / all be default. @codejedi365: if you prefer, I can pull them out; in my testing, removing them seems to be a noop.

Reply:
In this case, I'd rather just leave them as explicit defaults, especially since Ruff is so immature of a package that anything could change.